store/store.go

```diff
@@ -213,6 +290,16 @@ func NewStore(baseDir string) (*Store, error) {
 		if err = db.Do(fn); err != nil {
 			return nil, err
 		}
+
+		if err := s.storeLock.Close(); err != nil {
```

Comment:
Actually, this change looks strange. Are you fixing a bug that the lock was taken, but not released?